Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
```

Comment:
Since we have a `requirements.txt` file, can you remove the Pipfile and add it to the `.gitignore` please?